requirements.txt

```diff
@@ -13,16 +13,17 @@ html2text==2018.1.9
 Jinja2==2.10.1
 libsass==0.17.0
 lxml==3.7.1 ; sys_platform != 'win32' and python_version < '3.7'
-lxml==4.3.2 ; sys_platform != 'win32' and python_version >= '3.7'
+lxml==4.5.2 ; sys_platform != 'win32' and python_version >= '3.7'
 lxml ; sys_platform == 'win32'
 Mako==1.0.7
 MarkupSafe==1.1.0
 mock==2.0.0
 num2words==0.5.6
 ofxparse==0.19
 passlib==1.7.1
-Pillow==5.4.1 ; python_version < '3.7' or sys_platform != 'win32'
+Pillow==5.4.1 ; python_version < '3.7'
 Pillow==6.1.0 ; sys_platform == 'win32' and python_version >= '3.7'
+Pillow ; python_version > '3.7' and sys_platform != 'win32'
```

Comment:
What happens to those who have 3.7 and no win 32 ... I think that they won't have any Pillow package installed.